mass_mailing_partner/models/res_partner.py

```diff
@@ -14,7 +14,6 @@ class ResPartner(models.Model):
     mass_mailing_contact_ids = fields.One2many(
         string="Mailing contacts",
         oldname="mass_mailing_contacts",
-        domain=[('opt_out', '=', False)],
```

Comment:
Why this is needed to be removed?

Reply:
This not sure @chienandalu could you reply?